.github/actions/docker-logs/README.md

```diff
@@ -22,7 +22,7 @@ steps:
 - name: Start Cambpm
   uses: ./.github/actions/compose
   with:
-    compose_file: ${{ github.workspace }}/.github/actions/compose/docker-compose.cambpm.yml
+    .github/actions/compose/docker-compose.cambpm.yml
```

Comment:
I think you missed the property name here

Reply:
good catch!